dffml/distributed/todos.md

```diff
@@ -0,0 +1,12 @@
+## ToDo
+Onode waits till it has all the operations in the dataflow
+    - If there are multiple nodes having the same operation, but
+    for some reason their ack does not reach Onode before it has
+    obtained acks for all the operations, those worker nodes won't
+    be associated with the context.
```

Comment:
Please make sure we have a `README.md` in this directory that combines your nodes and todos and is up to date with the state of the distributed implementations, it's limitations, and reasonings for those limitations